### DISCUSSION TITLE
Wind Particle Animation Support

### DISCUSSION THREAD

**Discussion by yashp1998** [rnmapbox/maps#3622] [Ideas]
Mapbox has introduced a new feature for wind particle animation. You can view the demo here: https://docs.mapbox.com/mapbox-gl-js/example/raster-particle-layer.

It would be great to have support for a new property, such as a raster-array source and a raster-particle layer, to fully leverage this feature in custom implementations.

**Comment by sethjhs9** [comment]
Seconded, this would be a huge addition for my project. How do we typically add new sources/layers?

**Reply by sethjhs9** [reply]
Edit see https://github.com/rnmapbox/maps/discussions/3582#discussion-7015955
